x-pack/plugins/security_solution/public/common/components/sourcerer/index.test.tsx

```diff
@@ -677,617 +695,3 @@ describe('Sourcerer component', () => {
     expect(pollForSignalIndexMock).toHaveBeenCalledTimes(1);
   });
 });
-
```

Comment:
It was huge file and was causing weird error when changes some tests to RTL since single test suite shares the `jsdom` instance. So I have split this is not 5 files. One this + 4 new ones in the same folder